src/components/Thanks/MyKiva/OptInModule.vue

```diff
@@ -0,0 +1,247 @@
+import {
+	MOBILE_BREAKPOINT,
+} from '#src/composables/useBadgeModal';
+import KvUserAvatar from '@kiva/kv-components/vue/KvUserAvatar';
+import OptInNotification from './OptInNotification';
+
+const props = defineProps({
+	loans: {
+		type: Array,
+		default: () => ([])
+	},
+	optedIn: {
+		type: Boolean,
+		default: false
+	}
+});
+
+const apollo = inject('apollo');
+const newConsentAnswered = ref(false);
+const receiveNews = ref(false);
+
+const { isMobile } = useIsMobile(MOBILE_BREAKPOINT);
+
+const title = computed(() => {
+	if (props.optedIn) {
+		return 'Thank you! You reached a milestone';
+	}
+	if (props.loans.length === 1) {
+		return `Thank you! You and ${props.loans[0]?.name} are in this together now.`;
+	}
+	if (props.loans.length === 2) {
+		const names = props.loans.map(loan => loan?.name).join(' and ');
+		return `Thank you! You, ${names} are in this together now.`;
+	}
+
+	return `Thank you! You, ${props.loans[0]?.name}, ${props.loans[1]?.name}
+		and ${props.loans[2]?.name} are in this together now.`;
+});
+
+const description = computed(
+	() => `Want to hear how you're impacting ${props.loans[0]?.name}'s life and more ways to help people like them?`
+);
+
+const loansToDisplay = computed(() => props.loans.slice(0, 3));
+
+const getMarginRight = index => {
+	if (loansToDisplay.value.length > 2 && index === 0) {
+		if (isMobile.value) {
+			return '-81.5px';
+		}
+		return '-100px';
+	}
+
+	return '0';
+};
+
+const getMarginLeft = index => {
+	if (loansToDisplay.value.length > 1 && index === loansToDisplay.value.length - 1) {
+		if (loansToDisplay.value.length === 2) {
+			return '-63px';
+		}
+
+		if (isMobile.value) {
+			return '-81.5px';
+		}
+		return '-100px';
+	}
+
+	return '0';
+};
+
+const updateOptIn = value => {
+	if (value) {
+		try {
+			apollo.mutate({
+				mutation: gql`
+					mutation updateCommunicationSettings(
+						$lenderNews: Boolean
+					) {
+						my {
+							updateCommunicationSettings(
+								communicationSettings: {
+									lenderNews: $lenderNews
+								}
+							)
+						}
+					}
+				`,
+				variables: {
+					lenderNews: value,
+				},
+			});
+		} catch (error) {
+			logReadQueryError(error, 'OptInModule updateCommunicationSettings');
+		}
+	}
+	newConsentAnswered.value = true;
+	receiveNews.value = value;
+};
+</script>
+
+<style lang="postcss" scoped>
+
+.module-container {
+	max-width: 620px;
+	max-height: 900px;
+
+	@apply tw-flex tw-flex-col tw-mx-auto tw-overflow-hidden tw-opacity-full tw-bg-white
+		tw-text-center tw-px-3 md:tw-px-8 tw-gap-3 tw-rounded-lg tw-py-4;
+}
+
+.borrower-container {
+	animation: fadein ease-in 1s;
+	width: 70px;
+
+	@screen md {
+		width: 150px;
+	}
+
+	@apply tw-block tw-relative tw-mx-auto tw-z-4;
+}
+
+.borrower-container > div {
+	height: 100px;
+
+	@screen md {
+		height: 200px;
+	}
+}
+
+.borrower-image, .borrower-image :deep(img) {
+	width: 124px;
+	height: 124px;
+
+	@screen md {
+		width: 160px;
```

Comment:
Looking at Figma, the center borrower image should be `200px` even when there is a single loan, right now the single loan use case shows a smaller image at `160px`.